src/main/webapp/scripts/cookies.js

```diff
@@ -0,0 +1,85 @@
+const HOURS_PER_DAY = 24;
+const MINUTES_PER_HOUR = 60;
+const SECONDS_PER_MINUTE = 60;
+const SECONDS_PER_DAY = HOURS_PER_DAY*MINUTES_PER_HOUR*SECONDS_PER_MINUTE;
+const MILLISECONDS_PER_SECOND = 1000;
+
+/**
+ * Erase a cookie based on name.
+ */
+function eraseCookie(name) {
+  document.cookie = `${name}=;expires=Thu, 01 Jan 1970 00:00:00 UTC; path=/;`;
+}
+
+/**
+ * Create a new cookie based on name, value and days before expiring.
+ */
+function setCookie(name, value, expirationDays) {
+  let date = new Date();
+  date.setTime(date.getTime+(expirationDays*SECONDS_PER_DAY*MILLISECONDS_PER_SECOND));
+  document.cookie = `${name}=${value};expires=${expirationDays}`;
+}
+
+/**
+ * Set the cookie value of a change event.
+ */
+function setChangeCookie(event) {
+  let name = event.currentTarget.id;
+  let value = event.currentTarget.value;
+  setCookie(name, value, 1);
+}
+
+/**
+ * Set the cookie value of a checkbox click event.
+ */
+function setCheckboxCookie(event) {
+  let name = event.currentTarget.id;
+  let value = event.currentTarget.checked;
+  setCookie(name, value, 1);
+}
+
+
+/**
+ * Get the value of a cookie.
+ */
+function getCookie(name) {
```

Comment:
[Optional] I am used to seeing the methods defined in the order getX, setX, deleteX/eraseX since this is how most IDEs will autofill them.  There is no requirement that you move this method to the top but it makes a bit more sense in my head so I thought I'd add a suggestion in case you want to do that.

Reply:
I like that, I'll change it.